src/view/com/lists/ProfileLists.tsx

```diff
@@ -201,3 +218,7 @@ export const ProfileLists = React.forwardRef<SectionRef, ProfileListsProps>(
     )
   },
 )
+
+const styles = StyleSheet.create({
+  listsFooter: {paddingTop: 20},
```

Comment:
nit: is this worth creating separately for since it's only one style? Why not just inline it?

Reply:
That's how it's handled for feeds list - https://github.com/bluesky-social/social-app/blob/f68b15219fd02e23d965015201400138ed69d59d/src/view/com/posts/Feed.tsx#L575